apps/docs/joinTheme.config.js

```diff
@@ -0,0 +1,40 @@
+import { SEO_IMG_DEFAULT } from "@calcom/lib/constants";
+
+const themeConfig = {
+  titleSuffix: " | Join.com",
+  logo: <h4 className="m-0">Join.com</h4>,
+  head: (
+    <>
+      <meta name="msapplication-TileColor" content="#ffffff" />
+      <meta name="theme-color" content="#ffffff" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta httpEquiv="Content-Language" content="en" />
+      <meta
+        name="description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta
+        name="og:description"
+        content="Explore advice and explanations for all of our features, and discover new tips and tricks to get the most out of your subscription."
+      />
+      <meta name="og:title" content="Join Calendar" />
+      <meta name="og:image" content={SEO_IMG_DEFAULT} />
```

Comment:
common man, can you check line by line to remove everything that related to cal.com?  Or you are playing around with GPT?))

Reply:
I think that only thing that left is this line `<meta name="og:image" content={SEO_IMG_DEFAULT} />` 
but this var `SEO_IMG_DEFAULT` is pointing to `export const SEO_IMG_DEFAULT = `${WEBSITE_URL}/og-image.png`;` and it points to 
`export const WEBSITE_URL = process.env.NEXT_PUBLIC_WEBSITE_URL || "https://join.com";` so we can configure it throught our env variable and upload appropriate file to the server 🤔

Reply:
but we don't have access to logo. In this case if you will copy link to our website, picture will be broken. Better to remove it